### PR TITLE
Zegami tsv

### PR DESCRIPTION
**Pull request by strets123**

### CODE REVIEW

File: linear-tsv/index.md [hunk 1]
```diff
@@ -1,137 +1,56 @@
...
-A zero-length file is a valid linear TSV file, containing no records. Empty
-lines are ignored in linear TSV files.
-
-Linear TSV provides for escape sequences, instead of quoting, so that
-implementations can naively split on the byte values of the separators. To
-include newlines, tabs, carriage returns and backslashes in field data, the
-following escape sequences must be used:
-
-* `\n` for newline,
+Zegami expects there to be a header line across the top of the file. This
```
Comment (alightwing) [inline linear-tsv/index.md]: What happens in the frontend if a tsv is loaded with missing/empty header values? Language here is SHOULD, which implies optional, so valuable to communicate what happens if the thing is not as Zegami expects.

File: linear-tsv/index.md [hunk 2]
```diff
@@ -1,137 +1,56 @@
...
+should give names to each of the columns and should not contain empty values.
+Field names should not be repeated.
```
Comment (alightwing) [inline linear-tsv/index.md]: What happens in the frontend if the header names are repeated?

Comment (strets123) [reply linear-tsv/index.md]: They become multivalue fields

**Comment by ahaith** [comment]
Don't fully understand the context. Most tsvs are internal, only ZSC loads user-generated tsvs, and we don't need a rigorous spec for that.

**Pull request closed without merging**